Revit_Core_Adapter/Listener/Buttons/BHoMWebsite.cs

```diff
@@ -36,7 +37,8 @@ public class BHoMWebsite : IExternalCommand
 
         public Result Execute(ExternalCommandData commandData, ref string message, ElementSet elements)
         {
-            System.Diagnostics.Process.Start("https://bhom.xyz/");
+            var bHoMInformation = Query.Information();
+            System.Diagnostics.Process.Start($"{bHoMInformation.BHoMWebsite}");
```

Comment:
`Process.Start` is quite common so I would shorten it :bowtie: 
```suggestion
            Process.Start($"{bHoMInformation.BHoMWebsite}");
```

Reply:
also string interpolation is not needed here, can be as simple as `Process.Start(bHoMInformation.BHoMWebsite);`

Same in other calls